cmd/gogio/androidbuild.go

```diff
@@ -467,6 +470,13 @@ func exeAndroid(tmpDir string, tools *androidTools, bi *buildInfo, extraJars, pe
 				<category android:name="android.intent.category.LAUNCHER" />
 			</intent-filter>
 		</activity>
+		{{if .HasService}}
+		<service android:name="org.gioui.GioForegroundService"
+			android:stopWithTask="true">
+			<meta-data android:name="org.gioui.GioForegroundChannelID"
+			android:value="GioForegroundServiceChannel" />
```

Comment:
Nit: Drop "Channel" from the value, it seems redundant.

Also, is "org.gioui.ForegroundService" perhaps a better name?

Reply:
GioForegroundService just follows the same naming convention as GioActivity, GioApp, etc.

Reply:
GioForegroundService is ok as service class name; I was referring to the meta-data value. To be clear, I suggest:

```
<meta-data android:name="org.gioui.ForegroundChannelID" android:value="GioForegroundService"/>
```

Reply:
Or maybe even

<meta-data android:name="org.gioui.ForegroundChannelID" android:value="Foreground service"/>

in case the value ends up visible to the user (does it?) in the app settings.